client/src/outside/components/AssignmentsListing.vue

```diff
@@ -91,6 +96,14 @@ export default {
     showCreateDialog: function () {
       this.$refs.theDialog.showDialog()
     },
+    isItemMisconfigured (item) {
+      this.validationWarning = 'Outlined rows need further configuration. '
+      const DESCRIPTION = 'This assignment was automatically generated by an LTI request. Please update this description by editing the assignment in the EdEHR.'
+      const SEED_ID = '5e1dd3c8f3f225b4b6e937d8'
```

Comment:
Nope .. this id is a per mongo instance id and each tool consumer has it's own default seed.
The text can be found in the server ../src/config/text.js  
Create constants on the client side with the same text.  Add a comment in both the client and server to say these text strings need to be kept in sync.